.codeclimate.yml

```diff
@@ -24,5 +24,15 @@ ratings:
   - "**.go"
```

Comment:
Not specific to this line, but the section `ratings`. This has also been deprecated in version 2 and can be removed. All files are rated now by default.